test/test_errors.py

```diff
@@ -85,10 +85,10 @@ def test_handle_result_error(container_factory, rabbit_config):
             # proxy.task() will hang forever because it generates an error
             proxy.task.call_async()
 
-        with eventlet.Timeout(1):
```

Comment:
It turns out that `safety_interval` was also changed when we lifted this code. Kombu sets it to `1`, which explains the more-than-expected slowdown.

We've added a second to `QueueConsumer.stop`, which is used in about 120 tests. This is unfortunate but I don't think it justifies keeping this lifted code. I'm also concerned about the 10x difference in timeout interval -- I'm uncomfortable being so far away from value set/expected by kombu for no apparent reason.